sshh/main.py

```diff
@@ -110,10 +111,32 @@ def cmd_agent(request):
         sshenv['PS1'] = f"[{request.group}]{sshenv['PS1']}"
         logger.info('ssh-agent PID=%s session "%s" has been started. To close this session, exit shell.',
                     sshenv['SSH_AGENT_PID'], request.group)
-        subprocess.run (sshenv['SHELL'], env=sshenv)
+        shell_command = sshenv['SHELL']
+        bashrc_location = Path(sshenv.get('HOME', ''), '.bashrc')
+        if Path(shell_command).name.lower() == 'bash' and bashrc_location.is_file():
+            # set PS1 after bash execution if shell program is bash and if bashrc is found
+            temp_rc = mkstemp()
+            with open(temp_rc[1], 'w') as rc_file:
+                # Write temp rc file that sets PS1 after loading bashrc
+                rc_file.writelines([f'. {bashrc_location}\n',
+                                    f'PS1="{sshenv["PS1"]}"'])
```

Comment:
COULD for here,

PS1 might not be the only environment variable to be overridden. It might be valuable to put the entire sshenv contents in here, or at least the variables that SSHH is explicitly changing if there are any others.